debian/control

```diff
@@ -7,14 +7,14 @@ Standards-Version: 3.9.1
 Homepage: https://github.com/chipsec/chipsec
 
 Package: chipsec
-Architecture: any
+Architecture: amd64
```

Comment:
i think chipsec support x86 system too.

Reply:
Chipsec will run on x86 if I build it locally.  But when building debs so far, I've yet to get it to work. Even setting Architecture to All fails because at the end of the day, it's just installing a 64bit binary on a 32bit system (the binaries in chipsec_tools/linux are 64bit binaries).  This is the result of that attempt:

root@ubuntu-s-2vcpu-4gb-nyc3-01:~# chipsec_main
Traceback (most recent call last):
  File "/usr/bin/chipsec_main", line 9, in <module>
    load_entry_point('chipsec==1.3.5', 'console_scripts', 'chipsec_main')()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2569, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python2.7/dist-packages/chipsec_main.py", line 56, in <module>
    import chipsec.module
  File "/usr/lib/python2.7/dist-packages/chipsec/module.py", line 27, in <module>
    from chipsec.module_common import ModuleResult
  File "/usr/lib/python2.7/dist-packages/chipsec/module_common.py", line 43, in <module>
    import chipsec.chipset
  File "/usr/lib/python2.7/dist-packages/chipsec/chipset.py", line 41, in <module>
    from chipsec.helper.oshelper import OsHelper, OsHelperError
  File "/usr/lib/python2.7/dist-packages/chipsec/helper/oshelper.py", line 93, in <module>
    import chipsec.helper.helpers
  File "/usr/lib/python2.7/dist-packages/chipsec/helper/helpers.py", line 26, in <module>
    from chipsec.helper.linux import *
  File "/usr/lib/python2.7/dist-packages/chipsec/helper/linux/helper.py", line 56, in <module>
    from chipsec_tools import efi_compressor
ImportError: /usr/lib/python2.7/dist-packages/chipsec_tools/efi_compressor.so: wrong ELF class: ELFCLASS64

So maybe there's a way to fix that in deb packaging, but I'm no packaging expert, and to be honest, I don't really have the time to devote to fixing this at that level.  I do agree this is not the best solution though, so please do feel free to reject the PR.